src/coreclr/scripts/superpmi_collect_setup.py

```diff
@@ -39,22 +42,25 @@
 from coreclr_arguments import *
 from jitutil import run_command, copy_directory, copy_files, set_pipeline_variable, ChangeDir, TempDir
 
-
 # Start of parser object creation.
 
 parser = argparse.ArgumentParser(description="description")
 
-parser.add_argument("-source_directory", help="path to source directory")
```

Comment:
In addition to introducing `-payload_directory`, did you just move these around or are there any real updates too that I am not spotting?